### DISCUSSION TITLE
地图缩放层级限制和鼠标左键点击移动地图速度和移动的范围限制

### DISCUSSION THREAD

**Discussion by xiaobaicxk** [zouyaoji/vue-cesium#494] [Q&A] [answered]
大佬，请教一下框架里面如何实现地图缩放层级限制，没有在vc-viewer里面找到对应的参数设置
viewer.scene.screenSpaceCameraController.minimumZoomDistance = 1000;  
viewer.scene.screenSpaceCameraController.maximumZoomDistance = 30000;
还有框架里面是否有限制鼠标左键点击移动地图速度和移动的范围限制的方法或者对应的参数设置

**Comment by zouyaoji** [comment, marked as answer]
参考这个https://zouyaoji.top/vue-cesium/#/zh-CN/component/quickstart
拿到viewer就可以用Cesium API 设置了

**Reply by xiaobaicxk** [reply]
谢谢大佬
